nx/lib/nx.ex

```diff
@@ -3265,6 +3265,61 @@ defmodule Nx do
     end)
   end
 
+  @doc ~S"""
+  Split tensor or list into train and test subsets.
+
+  ## Examples
+
+    Split into 80% for training and 20% for testing.
+
+    iex> {_train, _test} = Nx.split(Nx.tensor([[3, 6, 5], [26, 75, 3], [23, 4, 1]]), split: 0.8)
+    {#Nx.Tensor<
```

Comment:
not sure what to do here to solve doctests. I think the problem is because I am returning a tuple.

Reply:
Just extract each tuple position to a variable and assert on each value on separate lines! I believe there are lots of examples on this in Nx.LinAlg